Jenkinsfile-e2e

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env groovy
+
+def dockerRegistry = "329802642264.dkr.ecr.eu-west-1.amazonaws.com"
+def nodeImageVersion = "10.16.0-1"
+def nodeImage = "${dockerRegistry}/bbc-news/node-10-lts:${nodeImageVersion}"
+
+def slackChannel = '#si_repo-simorgh'
+
+// Run latest every 3 hours, between 6 & 6, Monday - Friday
+def cron_string = env.BRANCH_NAME == "latest" ? "H 0 6-6/3 * * 1-5" : ""
+
+def notifySlack(messageParameters) {
+  def message = "${messageParameters.emoji} Simorgh E2Es ${messageParameters.buildStatus} on \"${messageParameters.branchName}\" [build #${env.BUILD_NUMBER}] ${env.BUILD_URL}"
+
+  slackSend(
+    channel: messageParameters.slackChannel,
+    color: messageParameters.colour,
+    message: message
+  )
+}
+
+pipeline {
+  agent any
+  triggers {
+      cron(cron_string)
+  }
+  options {
+    timeout(time: 90, unit: 'MINUTES')
```

Comment:
```suggestion
    timeout(time: 239, unit: 'MINUTES')
```

Reply:
To stop two running at once, we will fly past 90 very quickly.

Reply:
Can you clarify this `To stop two running at once`

Do you not want the tests against different environments run in parallel?